jerry-core/debugger/debugger.c

```diff
@@ -1487,21 +1485,21 @@ jerry_debugger_exception_object_to_string (ecma_value_t exception_obj_value) /**
   if (property_p == NULL
       || ECMA_PROPERTY_GET_TYPE (*property_p) != ECMA_PROPERTY_TYPE_NAMEDDATA)
   {
-    return ecma_new_ecma_string_from_utf8 (data, size);
+    return ecma_stringbuilder_finalize (&builder);
   }
 
   ecma_property_value_t *prop_value_p = ECMA_PROPERTY_VALUE_PTR (property_p);
 
   if (!ecma_is_value_string (prop_value_p->value))
   {
-    return ecma_new_ecma_string_from_utf8 (data, size);
+    return ecma_stringbuilder_finalize (&builder);
   }
 
-  data[size] = LIT_CHAR_COLON;
-  data[size + 1] = LIT_CHAR_SP;
+  ecma_stringbuilder_append_char (&builder, LIT_CHAR_COLON);
```

Comment:
`ecma_stringbuilder_append_char` is used to append `ecma_char_t`-s that need to be encoded, for ascii chars it's simpler to use `ecma_stringbuilder_append_byte`